relay-common/src/constants.rs

```diff
@@ -96,6 +96,10 @@ impl fmt::Display for EventType {
 #[repr(i8)]
 pub enum DataCategory {
     /// Reserved and unused.
```

Comment:
Where does "reserved and unused" come from? Do we have this enum variant because we started counting our data categories at 1 or because 0 actually did used to mean something? In the latter case I would be very careful about reusing this.

Reply:
Generally it does not matter as internal is not utilized. It's a reserved category. See other conversation.